restrict.dat.sas

```diff
@@ -1,18 +1,21 @@
 # Restricted frequency table for Saskatoon radar.
-default=13500
-8006 8016
-8071 8081
-8112 8122
-8162 8172
-9207 9217
-9476 9486
-9900 10100
-10191 10201
-12000 12995
-13355 13416
-14380 14410
-14676 14696
-14910 15090
-15729 15739
-15794 15804
-19985 20015
+# The use of frequencies in the following bands is not authorized: 9.995 - 10.005 MHz, 13.360 - 13.410 MHZ, 14.990 -
+# 15.010 MHZ, 19.990 - 20.010 MHZ
+# The use of the following frequencies is prohibited: 8.0115 MHz, 8.0765 MHz, 8.117 MHz, 8.1674 MHz, 9.2124 MHz,
+# 9.4815 MHz, 10.1965 MHz, 13.4115 MHz, 15.7345 MHz, 15.7995 MHz
+default=13500.0
+8011.5 8011.5
```

Comment:
If we want to keep the status quo, then just put a 6kHz band on either side of the 'prohibited' frequencies,
so for example this line would be :
```suggestion
8005.5 8017.5
```

Of course, we can decide if we want to put an extra band around these in Borealis, or directly in this file.
If we put it in Borealis, it could be more complicated based on what exactly we are transmitting, if we put it in this file then it would be simple but we would have to make it the max band we expect to transmit, probably not an issue unless someone wants a very specific frequency.